app.js

```diff
@@ -20,10 +20,20 @@ connect('http://127.0.0.1:8088', 'user', 'pass')
  */
 function clientLoaded (client) {
   client.start('sla');
-  sla(client)
-    .then(console.log)
-    .catch(errHandler)
-    .done();
+  var defer = Q.defer();
+  client.on('StasisStart', function(event, channel) {
+    stasisStart(event, client, channel);
+  });
+}
+
+function stasisStart (event, client, channel) {
+  if(event.args[0] !== 'dialed') {
```

Comment:
2. That is actually what I intend. If it has argument of 'dialed', then it should do nothing. Everything else gets a pass, (but it's checked if it's numeric later).